website/source/style/styles.css.scss

```diff
@@ -0,0 +1,221 @@
+@import "compass";
+@import "compass-placeholders";
+
+
+//// METRICS ////
+
+$grid: 4px;
+$column-width: 45 * $grid;
+$gap-width: 3 * $grid;
+
+
+//// COLORS ////
+
+@import "base/colors";
+@import "base/typography";
+
+
+//// SPRITES ////
+
+$graphics-spacing: 5 * $grid;
+@import "graphics/*.png";
+
+$graphics-2x-spacing: 10 * $grid;
+@import "graphics-2x/*.png";
+
+.graphics-sprite {
+  background-size: graphics-sprite-width(livingstyleguide);
+}
+
+
+//// MIXINS ////
+
+@mixin for-phones {
+  @media all and (max-width: 640px) {
+    @content;
+  }
+}
+
+@mixin for-phones-portrait {
+  @media all and (max-width: 480px) {
+    @content;
+  }
+}
+
+@mixin for-phones-landscape {
+  @media all and (min-width: 481px) and (max-width: 640px) {
+    @content;
+  }
+}
+
+@mixin for-tablets {
+  @media all and (min-width: 641px) and (max-width: 1024px) {
+    @content;
+  }
+}
+
+@mixin for-tablets-portrait {
+  @media all and (min-width: 641px) and (max-width: 768px) {
+    @content;
+  }
+}
+
+@mixin for-tablets-landscape {
+  @media all and (min-width: 769px) and (max-width: 1024px) {
+    @content;
+  }
+}
+
+@mixin for-tablets-landscape-and-desktops {
+  @media all and (min-width: 1024px) {
+    @content;
+  }
+}
+
+@mixin for-desktops {
+  @media all and (min-width: 1025px) {
+    @content;
+  }
+}
+
+@mixin for-hires {
+  @media only all and (-webkit-min-device-pixel-ratio: 1.3), only all and (min--moz-device-pixel-ratio: 1.3), only all and (-o-min-device-pixel-ratio: 1.3 / 1), only all and (min-device-pixel-ratio: 1.3), only all and (min-resolution: 1.3dppx) {
+    @content;
+  }
+}
+
+
+//// RESET EVERYTHING ////
+
+* {
+  border: none;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets